### DISCUSSION TITLE
We could benefit from beta testers

### DISCUSSION THREAD

**Discussion by DNin01** [ScratchAddons/ScratchAddons#6230] [Discussion]
Multiple times, Scratch Addons has had a moderately impactful bug after a feature update for one reason or another. Here are some examples:
- #6219
- #6062
- #4820

This wouldn't be as much of a problem if we had more beta testers trying out prereleases and reporting bugs before they make it to web stores. This is why software like Chrome and Edge have beta channels.

Related: #5490

**Comment by Samq64** [comment]
Do the extension stores support release channels, or will it need to be a separate extension?

**Reply by DNin01** [reply]
I'm not sure. If we want users to be able to install previews from web stores, we might have to publish that channel as a separate extension.

**Reply by WorldLanguages** [reply]
CWS recommends a separate extension item in the store labeled as [BETA] or similar. Still needs to go through review.

**Reply by Samq64** [reply]
Will reviews take as long as the stable version? Because if it does, that might not be a very practical solution if we want to release a new one every week or so.

**Reply by WorldLanguages** [reply]
> Will reviews take as long as the stable version?

Most likely yes.

**Comment by DNin01** [comment]
A similar all-in-one Scratch browser extension, ScratchTools, even has a whole [beta program](https://docs.scratchtools.app/docs/getting-started/installing-beta/testers). I'd love to be one of those people for Scratch Addons, reporting bugs before they affect the general public, hanging out with developers, etc. Though, that is already possible by installing from source and creating issues on GitHub. We just need more people doing it.
